src/diffuse/resources.py

```diff
@@ -75,10 +75,10 @@ def __init__(self):
         self.setKeyBinding('menu', 'go_to_line', defaultModKey + 'Shift+l')
         self.setKeyBinding('menu', 'realign_all', defaultModKey + 'l')
         self.setKeyBinding('menu', 'isolate', defaultModKey + 'i')
-        self.setKeyBinding('menu', 'first_difference', 'Shift+Ctrl+Up')
-        self.setKeyBinding('menu', 'previous_difference', 'Ctrl+Up')
-        self.setKeyBinding('menu', 'next_difference', 'Ctrl+Down')
-        self.setKeyBinding('menu', 'last_difference', 'Shift+Ctrl+Down')
+        self.setKeyBinding('menu', 'first_difference', defaultModKey + 'Shift+Up')
+        self.setKeyBinding('menu', 'previous_difference', defaultModKey + 'Up')
```

Comment:
> Ctrl + Up/Down is typically a system hotkey.

What in macOS is mapped to Ctrl + Up / Down? I could not see any conflicts myself.

I'd rather keep these 4 commands to the Ctrl button. When I created `defaultModKey`, I did not append it to these 4 bindings because, also on macOS, I felt it was more natural to use the Ctrl-key for everything "moving between differences"... Besides, just because we are on macOS, that doesn't mean we need to use the Command key for everything?

Compare with Visual Studio Code on macOS, for example. Its default keybindings ⌘↑ / ⌘↓ means "Go to beginning/end of file" but nothing is mapped to Ctrl + ↑ / ↓.

Source: https://code.visualstudio.com/shortcuts/keyboard-shortcuts-macos.pdf

Reply:
For "Mission Control" on my fairly recent M2 system:

<img width="631" alt="image" src="https://user-images.githubusercontent.com/1741643/198378074-72b0ed7a-abff-4a49-91fe-c677811dc224.png">

What OS version are you on?

I don't have a strong opinion about those shortcuts (and I can run a fork anyway), but other users with default settings will hit this exact problem.

Reply:
https://support.apple.com/guide/mac-help/open-windows-spaces-mission-control-mh35798/mac

Reply:
I see. Then it makes sense to use ⌘ instead, I guess.

Reply:
I'm willing to change <kbd>Ctrl</kbd> to <kbd>⌘</kbd> for the first/previous/next/last difference.

I do wonder though why not all the <kbd>Ctrl</kbd> here are not changed to <kbd>⌘</kbd>? It is my understanding that Mac doesn't really use <kbd>Ctrl</kbd>...

As Mac users, what is your experience with these bindings? The fact that some are using <kbd>Ctrl</kbd> and some are using <kbd>⌘</kbd>?

Reply:
Ctrl can be used, but perhaps mostly as a modifier to Cmd, which is the key that's used most frequently for commands.

Happy to address this in a new PR?

Reply:
Yep, I'll let you and @hugoholgersson (our two official Mac people on the project :wink: ) determine what would be the best modifiers to use on Mac.